lib/mobility/backend.rb

```diff
@@ -147,17 +146,6 @@ def setup &block
 
       def inherited(subclass)
         subclass.instance_variable_set(:@setup_block, @setup_block)
-        subclass.instance_variable_set(:@options, @options)
-        subclass.instance_variable_set(:@model_class, @model_class)
```

Comment:
`@options` and `@model_class` were being assigned to subclasses, but this doesn't really make sense since:
- "base" backend classes don't have these, so their subclasses are just assigning nil values
- configured backend classes don't (or shouldn't) have subclasses, so this would not be called anyway